study.md

```diff
@@ -152,7 +154,8 @@ What are the 4 most common HTTP verbs used when creating a RESTful API. How
 would you use each?
 
 ```md
-<!-- your answer here -->
+The 4 most common HTTP verbs are Get, Put, Delete and Post.
+
 ```
```

Comment:
Don't forget the second question, how would you use each, what are some general examples?